Dockerfile.testing

```
@@ -7,12 +7,18 @@ LABEL maintainer OSG Software <support@opensciencegrid.org>
 
 RUN \
     yum update -y && \
-    yum install -y --enablerepo=devops-itb osg-ca-generator && \
+    yum install -y git && \
```

```suggestion
    (yum install -y git-core || yum install -y git) && \
```
git-core is smaller but it's EL8 only.